lib/corebulk.go

```diff
@@ -306,9 +312,18 @@ func (b *BulkIndexer) Delete(index, _type, id string) {
 	return
 }
 
-func (b *BulkIndexer) UpdateWithWithScript(index string, _type string, id, parent, ttl string, date *time.Time, script string) error {
+func (b *BulkIndexer) UpdateWithWithScript(index string, _type string, id, parent, ttl string, date *time.Time, script EsScript, scripted_upsert bool, upsert interface{}) error {
```

Comment:
Is it still possible to supply an inline script?  Insofar as elastigo is a general purpose golang<->elasticsearch library, we shouldn't remove any functionality from it, even if we don't need that functionality ourselves.

Reply:
Ok I added the inline to the struct. The old function would actually not work with ES 2.X anyways: https://www.elastic.co/guide/en/elasticsearch/reference/2.3/modules-scripting.html